modules/oadp-checking-api-group-versions.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-checking-api-group-versions_{context}"]
+= Checking Kubernetes API group versions
```

Comment:
How about

> Listing the Kubernetes APIs available in the cluster

Reply:
@jboxman-rh -- How about "Listing the Kubernetes API group versions in a cluster" ?